modules/kubernetes-addons/kubernetes-dashboard/locals.tf

```diff
@@ -1,7 +1,8 @@
 locals {
   name                 = "kubernetes-dashboard"
   service_account_name = "eks-admin"
-  namespace            = "kube-system"
+  namespace            = local.name
```

Comment:
Moving k8s dashboard to a different namespace may lead into RBAC issues where the dashboard can't access k8s resources on other namespaces, I recall issues where if k8s dashboard isnt deployed on the kube-system then you need to deploy custom RBAC permissions to ensure it has access, since the default is kube-system.
If this has been tested and the k8s-dashboard can show/access etc. resources on all the other namespaces this is good